packages/wdio-cucumber-framework/src/utils.ts

```diff
@@ -198,6 +198,12 @@ export function addKeywordToStep(steps:ReporterStep[], feature:messages.GherkinD
             // the second from the TableRow AST node.
             // See https://github.com/cucumber/cucumber/blob/master/messages/messages.md
             const astNodeId = step.astNodeIds[0]
+
+            const rules  = feature.children.filter((child)=> Object.keys(child)[0]=== 'rule')
+            feature.children = feature.children.filter((child)=> Object.keys(child)[0]!== 'rule')
```

Comment:
```suggestion
            const rules  = feature.children.filter((child) => Object.keys(child)[0] === 'rule')
            feature.children = feature.children.filter((child) => Object.keys(child)[0] !== 'rule')
```

is `child` always an object with one property? Why do we always want to access the first key?

Reply:
First key is either rule or scenario always

Reply:
Makes sense! Thanks